Automattic/wp-calypso
# Remove Quick Start session mentions from /plans

#### Changes proposed in this Pull Request

* Per request in pc4ICr-mc-p2#comment-589, we're removing included Quick Start sessions for the eCommerce plan, so remove mentions of it from the /plans page.

#### Testing instructions



* Log in to WP.com and go to /plans.
* Verify that "Get personalized help" feature is not listed under any of the plans.
